src/test/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsCalculatorTest.java

```diff
@@ -20,91 +22,95 @@
     private static final LocalDate TO_DATE = LocalDate.of(2019, Month.JANUARY, 31);
     private static final LocalDate IN_RANGE = FROM_DATE.plusDays(2);
 
+    private static final List<AuditResultByNino> SOME_AUDIT_RESULTS = emptyList();
```

Comment:
Using "some" is a good way of indicating if the test value matters in the test. If it is a value that doesn't matter, it is better to use "any".